docs/modules/general/pages/implementation-details/mfe-content-loading/webcomponents.adoc

```diff
@@ -0,0 +1,195 @@
+
+**WebComponentLoaderModule** is a simple Angular module with a Router set up to always display **WebComponentLoaderComponent**. This component is responsible for rendering the html element required to display the module with Web Components Custom Elements. Whenever Shell UI's router detects change of the Microfrontend (another path is matched then the current one), it calls **loadChildren method** to load desired module and returns **WebComponentLoaderModule**. The module then creates and initializes **WebComponentLoaderComponent**, which after initializations creates the html element. Then Web Components using Custom Elements technology bounds the Microfrontend's module to the created element.
+
+For a detailed guide on how to expose Microfrontend's module via Webcomponent method, please refer here.
+
+// TODO: Add link to example of migrating angular remote components to integrate with OneCX
+=== Remote Component behavior
+Slot Component is responsible for displaying Microfrontend's Remote Components during Slot Component initialization. With the Webcomponent method, Slot Component will adapt the steps to:
+
+. load desired remote content via module-federation (no change)
+. create html element directly inside a container (instead of creating Microfrontend's Remote Component instance)
+
+The Slot Component content is based on https://angular.dev/api/core/ViewContainerRef[Angular ViewContainerRef] to dynamically create content inside the Slot Component. With Webcomponent method, the desired Microfrontend's Remote Component is still being loaded using module-federation. During that loading, it should register a component in Custom Elements Registry as mentioned in previous section. For OneCX platform it means that it no longer should create the Microfrontend's component directly, but ensure that the html code has correct element in order to display the component.
+
+The difference with the Webcomponent method is that, from the OneCX platform's perspective, no component needs to be created. Instead, a "host" element must be created to allow Web Components' Custom Elements to display desired content.
+
+For a detailed guide on how to expose Microfrontend's Remote Components via Webcomponent method, please refer here.
+
+== @onecx/angular-webcomponents details
+Using `@onecx/angular-webcomponents` to expose modules and Remote Components via Microfrontends is the recommended way within the OneCX platform. There are guides available that showcase how to transform already created Microfrontend's content to use Webcomponent method. In this section all of the speficic functions available within the angular-webcomponents library are explained.
+
+=== bootstrapModule function
+Use **bootstrapModule** function for module loading in the exposed file for a Microfrontend's module.
+
+This function ensures that module is bootstrapped correctly and takes care of:
+* ngZone sharing
```

Comment:
I think a newline is missing here, because it still is not showing it as bullet points (same for other parts below)